src/views/design/ScriptDetail/EditAction/ExpandableParameter.tsx

```diff
@@ -84,12 +85,16 @@ export default function ExpandableParameter({ parameter, onChange, constantParam
                     <InputLabel htmlFor="filled-adornment-password">
                         <Translate msg="scripts.detail.edit_action.parameter.value" />
                     </InputLabel>
-                    <FilledInput
+                    <TextInput
                         id={`${constantParameter.name}-input`}
                         type="text"
                         value={parameter ? parameter.value : ''}
                         onChange={(e) => onChange(e.target.value)}
                         multiline
+                        InputProps={{
+                            readOnly: !checkAuthority(SECURITY_PRIVILEGES.S_SCRIPTS_WRITE, 'PUBLIC'),
```

Comment:
Can we already use the security group name  of the script in question instead of the fixed value 'PUBLIC'